### PR TITLE
Update Create-Aleo-App

### PR DESCRIPTION
**Pull request by oguzutku1745**
Installation commands updated depends on the current command usage. I created 2 PR to update some of the templates:

https://github.com/AleoHQ/sdk/pull/880
https://github.com/AleoHQ/sdk/pull/879

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207152764972226

### CODE REVIEW

File: src/build/leo/aleo-sdk/create-aleo-app/installation.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+   - `Vanilla (JavaScript)`
+
+3. If you choose `React` as your framework, the supported templates are:
+   - `JavaScript` + `Leo`
+   - `TypeScript` + `Leo`
+   - `TypeScript` + `Next.js`
+     
+## More Information
+
+Based off of create-vite: https://github.com/vitejs/vite/tree/main/packages/create-vite
```
Comment (ramidecodes) [inline src/build/leo/aleo-sdk/create-aleo-app/installation.md:32]: Should this read "Based on `create-vite`" ??

**Review by ramidecodes — APPROVED** [review]

File: src/build/leo/aleo-sdk/create-aleo-app/installation.md (hunk 1 above)
Comment (oguzutku1745) [reply src/build/leo/aleo-sdk/create-aleo-app/installation.md:32]: This is from the current documentation. The best example is on Vite. I think that is the reason they highlighted `vite` at the end, but we also have the rest. Do you think it should remain as it is, or should I add the other frameworks too?

Comment (oguzutku1745) [reply src/build/leo/aleo-sdk/create-aleo-app/installation.md]: You are right, I converted it to "Based on create-vite"

**Review by zklim — APPROVED** [review]
lgtm

**Comment by oguzutku1745** [comment]
> @oguzutku1745 Please resolve the conversations when you've updated

Oh, I forgot it. Thanks for reminding!

**Pull request merged**